plugins/importexport/native/NativeImportDom.inc.php

```diff
@@ -813,6 +813,10 @@ function handleArticleNode(&$journal, &$articleNode, &$issue, &$section, &$artic
 		}
 		if ($hasErrors) return false;
 		
+		$citationDao =& DAORegistry::getDAO('CitationDAO');
```

Comment:
Yes, I have tested (WAMP with PHP 5.4) both Import, Export Issue and Import
Article
From report https://travis-ci.org/pkp/ojs/jobs/133348442 I assumed that PHP
5.3 did fail that.
No sweet, I will tempt import some archive issues next week to online ojs
and tell.
Thank You


*С уважением,Игорь Перескоков,*
*администратор сайтов,*
*ТПГ "ПетроСити"*

*м.т. +7-931-288-16-79*
| www.petrocity.ru | nskenergo.com | 21mm.ru | 21bs.ru | mmps.ru |
pramac-spb.ru | teplocity.spb.ru | poltavamash.su

2016-05-27 15:53 GMT+03:00 Alec Smecher <notifications@github.com>:

> In plugins/importexport/native/NativeImportDom.inc.php
> <https://github.com/pkp/ojs/pull/853#discussion_r64899653>:
>
> > @@ -813,6 +813,10 @@ function handleArticleNode(&$journal, &$articleNode, &$issue, &$section, &$artic
> >  		}
> >  		if ($hasErrors) return false;
> >  		
> > +		$citationDao =& DAORegistry::getDAO('CitationDAO');
>
> Have you tested an import? The tool we use for continuous integration
> tests fails on the import test, and it's probably because of this section,
> I think.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pkp/ojs/pull/853/files/44fda891526fa8003f7cb213b095c61499117987#r64899653>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASsrb_QOM72nZGWjPUs3PlxbhSXwPE8pks5qFulMgaJpZM4IoY9E>
> .
>